src/vic3_world/buildings/building_importer.h

```diff
@@ -26,10 +27,12 @@ class BuildingImporter
    std::string type_;
    std::optional<int> state_number_;
    float goods_sales_value_;
+   float staffing_level_;
+   std::vector<std::string> production_methods_;
```

Comment:
Expand Vic3WorldBuildingsBuildingImporterTests::DefaultsAreDefaulted to cover these members.

Reply:
Done.